tuplex/test/wrappers/WrapperTest.cc

```diff
@@ -2395,6 +2393,69 @@ TEST_F(WrapperTest, NYC311) {
     }
 }
 
+TEST_F(WrapperTest, MixedTypesIsWithNone) {
+    using namespace tuplex;
+    using namespace std;
+
+    PythonContext c("python", "",  "{\"tuplex.webui.enable\":\"False\", \"tuplex.optimizer.mergeExceptionsInOrder\":\"True\"}");
+
+    // use this test data as soon as parallelize(...) supports exception model properly...
```

Comment:
do we have a test testing the commented code? If yes, remove commented code. Else, maybe create a new test with some example of each type basically (ultimate mix test).